### PR TITLE
Simple previews

### PR DESCRIPTION
**Pull request by micapam**
Loads a preview inline from the server and pushes it onto the page. 

Ugly as all hell but working - except that chart styling isn't happening (numbers show that the preview is working) ...

### CODE REVIEW

**Comment by elisechant** [comment]
So as discussed, we should make this an iframe. And we'll need to rework some of the React parts to be more React-land. But WIP.

**Comment by micapam** [comment]
Ready for reviews. The JS code is nasty. But - once the hero preview is fixed - could work as an MVP for user testing.

File: app/decorators/previewing_dataset_decorator.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class PreviewingDatasetDecorator < Draper::Decorator
```
Comment (micapam) [inline app/decorators/previewing_dataset_decorator.rb]: Good point

File: app/views/dashboards/export.csv.shaper [hunk 2]
```diff
@@ -0,0 +1,15 @@
+csv.headers :dataset_name, :units, :time_stamp, :label, :value
```
Comment (tobyhede) [inline app/views/dashboards/export.csv.shaper]: Not seen this gem before, nice one.

**Comment by tobyhede** [comment]
Looks good, I'd add a spec for the decorator, even as mvp

**Comment by micapam** [comment]
Spec has decorator now @tobyhede

**Comment by tobyhede** [comment]
LGTM 👍

**Comment by elisechant** [comment]
cool, but we're going to need to do a version 2 to improve some of the front end codes

**Comment by elisechant** [comment]
created https://govausites.atlassian.net/browse/PD-930

**Pull request merged**